app/controllers/group_members_controller.rb

```diff
@@ -29,6 +29,7 @@ def set_group
   end
 
   def group_member_params
-    params.require(:group_member_form).permit(:member_email_address).merge(group: @group, creator: current_user)
+    ## TODO: We are passing in host here because the admin doesn't know it's own URL to use in emails
```

Comment:
This is a TODO but I'm not entirely sure what needs to be done - is there a card that can be referenced here?

Reply:
yes, I should make one.

It's related to the point we discussed about the admin's URL in tech catchup on monday: https://docs.google.com/document/d/1iHfUoCzgg-2Mk9wbi5o46fFVRak_R6nNQ2m8OqSz_hs/edit

Reply:
https://trello.com/c/zSQq9xee/1773-configure-admin-to-use-url-from-env-in-email-config